Tests/Acceptance/ClearRevokedTokensCommandTest.php

```diff
@@ -0,0 +1,184 @@
+        );
+    }
+
+    public function testClearRevokedAccessAndRefreshTokensAndAuthCodes(): void
+    {
+        $this->assertNotNull(
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_REVOKED
+            )
+        );
+        $this->assertNotNull(
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_REVOKED
+            )
+        );
+        $this->assertNotNull(
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_REVOKED
+            )
+        );
+
+        $output = $this->executeCommand();
+
+        $this->assertStringContainsString('Access tokens deleted: 1.', $output);
+        $this->assertStringContainsString('Refresh tokens deleted: 1.', $output);
+        $this->assertStringContainsString('Auth codes deleted: 1.', $output);
+
+        $this->assertNull(
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_REVOKED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_REVOKED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_REVOKED
+            )
+        );
+    }
+
+    public function testClearRevokedAccessTokens(): void
+    {
+        $output = $this->executeCommand([
+            '--access-tokens' => true,
+        ]);
+
+        $this->assertStringContainsString('Access tokens deleted: 1.', $output);
+        $this->assertStringNotContainsString('Refresh tokens deleted', $output);
+        $this->assertStringNotContainsString('Auth codes deleted', $output);
+
+        $this->assertNull(
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_REVOKED
+            )
+        );
+        $this->assertInstanceOf(
+            RefreshToken::class,
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_REVOKED
+            )
+        );
+        $this->assertInstanceOf(
+            AuthorizationCode::class,
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_REVOKED
+            )
+        );
+    }
+
+    public function testClearRevokedRefreshTokens(): void
+    {
+        $output = $this->executeCommand([
```

Comment:
@franjo-zadelj-trikoder The method is called `executeCommand`, yet it clears the entity manager as well, the two seem unrelated. Perhaps it'd be better to call `clearEntityManager` explicitly after `executeCommand`.

Reply:
@HypeMC maybe rename it to "executeCommandAndClear"? I'm trying to avoid unnecessary code duplication if possible.

Reply:
@HypeMC done